### ISSUE TITLE
(Testbed) Box position offset when not using piqnt editor

### ISSUE THREAD

**Issue by armput** [piqnt/planck.js#295]
I'm learning planck and I'm using the AIJS Editor and whenever I run the testbed, it offsets boxes by about (-0.3, 0.3). 
![image](https://github.com/user-attachments/assets/3df3116f-5308-4abe-baa1-e533fb692018)
The dot and the body are both at (0, 0) but you can see that the `box` is offset a tiny bit.

Here's the code:
```
const { World, Testbed, Box, Vec2 } = planck;
planck.testbed(function(testbed) {
    const world = new World();

    let box = world.createDynamicBody(Vec2(0, 0));
    box.createFixture(Box(2, 8), 2);
    
    console.log(box.getPosition());

    testbed.step = function() {
        let pos = box.getPosition();
        testbed.drawCircle({x: pos.x, y: pos.y}, 0, '#fff');
    }

    return world;
});
```

I went over to the piqnt.com editor and it works as expected but in the other editors I've tried boxes are offset.

Is there any way to fix this issue more permanently than offsetting the position?

[Update]: So apparently this also applies to polygons.

**Comment by armput** [comment]
`planck.testbed(function(testbed) {
    const world = new World();

    let box = world.createDynamicBody(Vector(0, 0));
    box.createFixture(Box(2, 8), 2);
    
    console.log(crank.getPosition());

    testbed.step = function() {
        let pos = crank.getPosition();
        testbed.drawCircle({x: pos.x, y: pos.y}, 0, '#fff');
    }

    return world;
});`

**Comment by armput** [comment]
(Update) This only seems to be happening with `Boxes`, I tried the other ones and they worked fine.

**Comment by shakiba** [comment]
Thanks for reporting!

I'm rewriting the testbed, I will fix this one as well.

Currently the testbed in this repo is slightly different from the implementation in piqnt.com, after rewrite they will share the same code.

**Comment by shakiba** [comment]
I fixed this in v1.3.0.
